src/core/PlayArea.ttslua

```diff
@@ -504,20 +504,6 @@ function shiftContents(playerColor, direction)
   Wait.time(drawBaseConnections, 0.1)
 end
 
--- Returns the current value of the investigator counter from the playmat
----@return. Number of investigators currently set on the counter
-function getInvestigatorCount()
-  local investigatorCounter = getObjectFromGUID("f182ee")
-  return investigatorCounter.getVar("val")
-end
-
--- Updates the current value of the investigator counter from the playmat
----@param count Number of investigators to set on the counter
-function setInvestigatorCount(count)
-  local investigatorCounter = getObjectFromGUID("f182ee")
-  return investigatorCounter.call("updateVal", count)
-end
-
```

Comment:
Can you confirm that you checked whether this is referenced by any other scripts?

Reply:
Yep!  It only existed for the API, so I just put it in there for clarity